prompt_toolkit/shortcuts/dialogs.py

```diff
@@ -175,6 +175,7 @@ def radiolist_dialog(
     ok_text: str = "Ok",
     cancel_text: str = "Cancel",
     values: Optional[List[Tuple[_T, AnyFormattedText]]] = None,
+    default_values: List[_T] = [],
```

Comment:
mutable values should not be used as keyword arguments. State could leak from one call to the next.

Better is to do:
```python
default_values: Optional[List[_T]] = None,
```

and in the body of the function:
```python
if default_values is None:
    default_values = []
```